src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests/Attributes/SetupAccountIdAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AutoFixture;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests.SpecimenBuilders;
+
+namespace SFA.DAS.ProviderPayments.Calc.Shared.IntegrationTests.Attributes
+{
+    public class SetupAccountIdAttribute : TestActionAttribute
+    {
+        public override ActionTargets Targets { get; } = ActionTargets.Suite;
+
+        public override void BeforeTest(ITest test)
+        {
+            var fixture = new Fixture();
+            fixture.Customizations.Add(new KnownAccountIdBuilder());
+            
+            SharedTestContext.AccountId = fixture.Create<long>();
```

Comment:
I'm not a fan of using statics. Carl mentioned to me that there is a NUnitContext which may be better to use (but we can do that later).